[2024-06-11T22:41:13.419+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task3 manual__2024-06-11T22:41:09.627782+00:00 [queued]>
[2024-06-11T22:41:13.424+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task3 manual__2024-06-11T22:41:09.627782+00:00 [queued]>
[2024-06-11T22:41:13.424+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T22:41:13.424+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-11T22:41:13.425+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T22:41:13.431+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task3> on 2024-06-11 22:41:09.627782+00:00
[2024-06-11T22:41:13.434+0000] {standard_task_runner.py:55} INFO - Started process 3811 to run task
[2024-06-11T22:41:13.436+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task3', 'manual__2024-06-11T22:41:09.627782+00:00', '--job-id', '230', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmpzydjfjtn']
[2024-06-11T22:41:13.436+0000] {standard_task_runner.py:83} INFO - Job 230: Subtask task3
[2024-06-11T22:41:13.466+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task3 manual__2024-06-11T22:41:09.627782+00:00 [running]> on host 91b1d49ba724
[2024-06-11T22:41:13.498+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task3
AIRFLOW_CTX_EXECUTION_DATE=2024-06-11T22:41:09.627782+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-11T22:41:09.627782+00:00
[2024-06-11T22:41:13.499+0000] {logging_mixin.py:120} INFO - Task 3 is completed
[2024-06-11T22:41:14.500+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-11T22:41:14.507+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task3, execution_date=20240611T224109, start_date=20240611T224113, end_date=20240611T224114
[2024-06-11T22:41:14.549+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-11T22:41:14.563+0000] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
